keylime/tenant.py

```diff
@@ -692,16 +692,17 @@ def do_cvbulkinfo(self):
 
         response_json = Tenant._jsonify_response(response, print_response=False)
 
+        if response.status_code == 500:
+            raise UserError(f"Unexpected response from Verifier {self.verifier_ip} on port {self.verifier_port}")
```

Comment:
I looks like the logger.info at the bottom (line 725) should probably be converted to a catch-all that includes this 500...

Reply:
yup, done. Went "minimalistic" on the response code processing instead